hpOneView/resources/servers/server_profiles.py

```diff
@@ -333,8 +229,8 @@ def get_available_networks(self, **kwargs):
         Returns:
             list: Available networks.
         """
-        uri = self.__build_uri_with_query_string(kwargs, '/available-networks')
-        return self._client.get(uri)
+        uri = self._helper.build_uri_with_query_string(kwargs, '/available-networks')
+        return self._helper.do_get(uri)
 
     def get_available_servers(self, **kwargs):
         """
```

Comment:
we need to add the filter query param in the comments section

Reply:
Added